druntime/src/__builtins.di

```diff
@@ -53,6 +53,10 @@ version (CRuntime_Microsoft)
 
 version (DigitalMars)
 {
+    immutable float __nan = float.nan;
+
+    float __builtin_nanf()(char*)  { return float.nan; }
```

Comment:
in which case this should probably be 
`float __builtin_nanf()(const char* p) { assert (!p); return float.nan; }`

Reply:
The use cases I've seen used a non-null string to specify an ordinary nan. So the assert would always trip.